diff --git a/node_modules/@apidevtools/json-schema-ref-parser/lib/util/url.js b/node_modules/@apidevtools/json-schema-ref-parser/lib/util/url.js
index 81b3a0d..5449bf1 100644
--- a/node_modules/@apidevtools/json-schema-ref-parser/lib/util/url.js
+++ b/node_modules/@apidevtools/json-schema-ref-parser/lib/util/url.js
@@ -1,5 +1,6 @@
 "use strict";
 
+let process = require("process");
 let isWindows = /^win/.test(process.platform),
     forwardSlashPattern = /\//g,
     protocolPattern = /^(\w{2,}):\/\//i,
